processor/configuration/src/main/resources/schemas/actions/kafka_topic_sink_0.1.json

```diff
@@ -7,9 +7,30 @@
       "title": "Topic Name",
       "description": "The topic where to send the event.",
       "example": "my-topic"
+    },
+    "kafka_broker_url": {
+      "type": "string",
+      "title": "Broker URL",
+      "description": "RHOSAK Broker URL ",
+      "example": "username-c--hj---mhlksdfss-p--a.bf2.kafka.rhcloud.com:443"
+    },
+    "kafka_client_id": {
+      "type": "string",
+      "title": "Client Id",
+      "description": "The Client Id part of the credentials to authenticate to Kafka",
+      "example": ""
```

Comment:
If there's no value, the `example` property should be removed. Otherwise it will show up as `Example: ` below the input field.

Same for `kafa_client_secret`.